MarathonRecomp/patches/fps_patches.cpp

```diff
@@ -121,3 +123,29 @@ void PlayerObject_ProcessMsgSuckPlayer_FixDeltaTime(PPCRegister& message, PPCReg
 
     deltaTime.f64 = pMessage->DeltaTime;
 }
+//not need anything here
+void ParticleHFR_826454C8(PPCRegister& f31)
```

Comment:
If this patch is just being used as a NOP, then please remove this function and use the `NOP` one.